model/tools/ftn2src.sh

```diff
@@ -0,0 +1,53 @@
+
+thisdir=`pwd`
+ftndir="$thisdir/../ftn"
+srcdir="$ftndir/../src"
+
+mkdir $srcdir
+
+DIRLIST="SCRIP PDLIB ." 
+
+for DIR in $DIRLIST
+do 
+
+  cd $ftndir/$DIR
+  ftnfiles=`ls *.ftn`
+  ftnfiles=$(sed -e "s/.ftn//g" <<< $ftnfiles)
+  nonftnfiles=`ls -I "*.ftn"`
+ 
+  cd $thisdir
+  if [ ! -d $srcdir/$DIR ] 
+  then 
+     mkdir -p $srcdir/$DIR
+  fi  
+
+  set -x
```

Comment:
Can we remove this? It makes it very verbose.

Reply:
Yes - I forgot to remove it after debugging some things.